images/win/scripts/Installers/Install-Stack.ps1

```diff
@@ -19,4 +19,6 @@ Extract-7Zip -Path $StackArchivePath -DestinationPath $DestinationPath
 
 New-Item -Name "x64.complete" -Path $StackToolcachePath
 
-Add-MachinePathItem -PathItem $DestinationPath
\ No newline at end of file
+Add-MachinePathItem -PathItem $DestinationPath
+
+Invoke-PesterTests -TestFile "Stack.Tests"
```

Comment:
`Invoke-PesterTests -TestFile "Stack" ` just "Stack".
".Tests.ps1" will be added automatically

Reply:
The same for other tools